src/proto/cloudevents/protocloudevent.rs

```diff
@@ -23,6 +23,58 @@ use crate::cloudevents::cloud_event::CloudEventAttributeValue;
 use crate::cloudevents::cloud_event::Data as CloudEventData;
 use crate::cloudevents::CloudEvent as CloudEventProto;
 
+use crate::uprotocol::umessage::UMessage;
+use crate::uri::serializer::{LongUriSerializer, UriSerializer};
+
+impl From<UMessage> for cloudevents::Event {
+    fn from(source_event: UMessage) -> Self {
+        let source = source_event.source.get_or_default();
```

Comment:
I noticed leaning on getting default and unwraps throughout.

I'm wondering if we should be doing any additional error checking and make it TryFrom instead to reflect that it may fail?

WDYT?

Reply:
we have a try_from now - However, next I will have to spend some quality time revisiting the test setup for cloudevent.rs, stay tuned for another change in that area.